### PR TITLE
Feat/sc 75 typography update

### PR DESCRIPTION
**Pull request by davidgilbertson**
Jira: https://govausites.atlassian.net/browse/SC-75

I've updated the readme, should be self explanatory (if not I'll update it more).

It's a bit of a weird one since none of the designs actually have the correct font styles, so there's nothing to check against.

### CODE REVIEW

File: src/components/AllCitiesOverviewContainer/AllCitiesOverview/AllCitiesOverview.scss [hunk 1]
```diff
@@ -156,15 +151,17 @@ $minister-image-width: 214px;
...
 }
 
 .subInfoBodyText,
 .infoBodyText {
-  font-size: 22px;
+  font-size: 20px;
+  font-weight: 300;
   line-height: 37px;
-  letter-spacing: -0.7px;
+  letter-spacing: 1px;
```
Comment (micapam) [inline src/components/AllCitiesOverviewContainer/AllCitiesOverview/AllCitiesOverview.scss:164]: The letters look way too widely spaced to me. 

image

**Review by micapam — CHANGES_REQUESTED** [review]
One style needs fixing, otherwise good to merge

File: src/components/AllCitiesOverviewContainer/AllCitiesOverview/AllCitiesOverview.scss (hunk 1 above)
Comment (davidgilbertson) [reply src/components/AllCitiesOverviewContainer/AllCitiesOverview/AllCitiesOverview.scss:164]: What browser is that? It doesn't look right.

Here's how it looks for me, Chrome on Mac and Edge and Chrome on Windows all looked fine. The designs are overlaid (the lighter text)

image

Comment (micapam) [reply src/components/AllCitiesOverviewContainer/AllCitiesOverview/AllCitiesOverview.scss:164]: Chrome 61 on Mac.

Comment (davidgilbertson) [reply src/components/AllCitiesOverviewContainer/AllCitiesOverview/AllCitiesOverview.scss:164]: Can you screenshot the whole screen? This is Chrome 61 macOS for me:
image

**Review by micapam — APPROVED** [review]

**Pull request merged**